### DISCUSSION TITLE
Issue with Publish test results to Alva / tests

### DISCUSSION THREAD

**Discussion by abhikmr778** [DevSkillsHQ/discusssions#35] [Help] [answered]
Hi all

My tests are passing, but the results are not getting published. 
I am getting the following error.

Run retry=0
/home/runner/work/_temp/2c263d63-ec92-45e6-b322-d3af7cf130b2.sh: line 25: unexpected EOF while looking for matching `"'
Error: Process completed with exit code 2.

I am not sure if this is an issue from my side.
![Screenshot (47)](https://github.com/DevSkillsHQ/discusssions/assets/26161187/831fea41-4241-4aff-a28c-062de141c8c0)

Please help me with the above :)

Best Regards
Abhinav

**Comment by fenske** [comment, marked as answer]
Hi @abhikmr778. I'm Anton from Alva. Sorry about this issue. We had a little bug that didn't escape the JSON body, causing this. Fixed now. And your results have been successfully shared with the hiring team.

I wish you the best of luck with the rest of your interview process!

**Reply by abhikmr778** [reply]
Great, thanks a lot!
